src/CmsChangeLogBundle/src/esono/pkgcmschangelog/TCMSTableEditor/TCMSTableEditorChangeLog.class.php

```diff
@@ -62,13 +63,20 @@ public function Save(&$postData, $bDataIsInSQLForm = false)
     {
         if ($this->oTableConf->fieldChangelogActive) {
             $this->failOnForbiddenTables();
-            $this->savePreSaveValues($postData);
+
+            if (isset($postData['id'])) {
+                $this->bIsUpdate = true;
+            } else {
+                $this->bIsUpdate = false;
+            }
         }
 
         return parent::Save($postData, $bDataIsInSQLForm);
     }
 
     /**
+     * Not used anymore. Handled with $this->oTablePreChangeData.
```

Comment:
Could you file an issue for deprecating this method in 6.3 (or file a PR directly - such a simple deprecation doesn't need to be tested, so we can omit the issue)?

Reply:
How would I do this?

Such a PR against 6.3 would still need be based on this branch?

Reply:
Just leave a note in the PR description and refer to this issue, so we can merge it after the changes from this PR are merged to master.